http/vibe/http/common.d

```diff
@@ -398,11 +399,17 @@ final class ChunkedInputStream : InputStream
 		readChunk();
```

Comment:
It probably makes sense to take the opportunity and remove this as well, so that the stream is completely lazy.

Reply:
```suggestion
```